lib/get-vendor-prefix.js

```diff
@@ -6,7 +6,7 @@ Object.defineProperty(exports, '__esModule', {
 exports['default'] = getVendorPrefix;
 
 function getVendorPrefix(prop) {
-  if (!document) return prop;
+  if (typeof document == 'undefined') return prop;
```

Comment:
better to check `document === undefined`

See lodash's implementation for reference.